frontends/election-manager/src/lib/backends/admin_backend.test.ts

```diff
@@ -145,24 +146,112 @@ test('addCastVoteRecordFile happy path', async () => {
   await storage.set(currentElectionIdStorageKey, 'test-election-2');
   fetchMock.get('/admin/elections', { body: getElectionsResponse });
 
+  const exportedTimestamp = '2021-09-02T22:27:58.327Z';
   const apiResponse: Admin.PostCvrFileResponse = {
     status: 'ok',
     id: 'cvr-file-1',
     alreadyPresent: 10,
+    exportedTimestamp,
+    fileMode: Admin.CvrFileMode.Test,
+    fileName: 'cvrs.jsonl',
     newlyAdded: 450,
+    scannerIds: ['scanner-4', 'scanner-6'],
     wasExistingFile: false,
   };
-  fetchMock.post(`/admin/elections/test-election-2/cvr-files?`, apiResponse);
+  fetchMock.post(
+    '/admin/elections/test-election-2/cvr-files?',
+    (_url, mockRequest) => {
+      assert(mockRequest.body instanceof FormData);
+
+      const formData: FormData = mockRequest.body;
+      expect(formData.get('exportedTimestamp')).toBe(exportedTimestamp);
+
+      return apiResponse;
+    }
+  );
 
   await expect(backend.loadCastVoteRecordFiles()).resolves.toBeUndefined();
 
   await expect(
     backend.addCastVoteRecordFile(
-      new File([partial1CvrFile.asBuffer()], 'cvrs.jsonl')
+      new File([partial1CvrFile.asBuffer()], 'cvrs.jsonl', {
+        lastModified: new Date(exportedTimestamp).valueOf(),
+      })
+    )
+  ).resolves.toEqual<Admin.CvrFileImportInfo>({
```

Comment:
TIL `toEqual` has a type param. Now to replace all `toEqual(typedAs<T>(…))` with `toEqual<T>(…)`.

Reply:
ha! honestly, same